src/elements/slider/slider.visual.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { html, nothing, type TemplateResult } from 'lit';
+
+import { describeViewports, visualDiffDefault, visualDiffFocus } from '../core/testing/private.js';
+
+import '../form-field.js';
+import './slider.js';
+
+describe('sbb-slider', () => {
+  const defaultArgs = {
+    disabled: false,
+    readonly: false,
+    withIcon: true,
+    withSlottedIcon: false,
+  };
+
+  const standaloneTemplate = ({
+    disabled,
+    readonly,
+    withIcon,
+    withSlottedIcon,
+  }: typeof defaultArgs): TemplateResult => html`
+    <sbb-slider
+      ?disabled=${disabled}
+      ?readonly=${readonly}
+      start-icon=${withIcon && !withSlottedIcon ? 'walk-slow-small' : nothing}
+      end-icon=${withIcon && !withSlottedIcon ? 'walk-fast-small' : nothing}
+    >
+      ${withIcon && withSlottedIcon
+        ? html`
+            <sbb-icon slot="prefix" name="battery-level-empty-small"></sbb-icon>
+            <sbb-icon slot="suffix" name="battery-level-high-small"></sbb-icon>
+          `
+        : nothing}
+    </sbb-slider>
+  `;
+
+  const formFieldTemplate = (args: typeof defaultArgs): TemplateResult => html`
+    <sbb-form-field>
+      <label>Label</label>
+      ${standaloneTemplate(args)}
+    </sbb-form-field>
+  `;
+
+  const variants = [
+    { name: 'standalone', template: standaloneTemplate },
+    { name: 'form-field', template: formFieldTemplate },
+  ];
+
+  describeViewports({ viewports: ['zero', 'medium'] }, () => {
+    for (const variant of variants) {
+      for (const visualDiffState of [visualDiffDefault, visualDiffFocus]) {
+        it(
```

Comment:
Since it's 5 tests instead of inlining variant.name each time you could wrap them in a describe block, what do you think?

Reply:
it should be fixed now